pontoon/base/views.py

```diff
@@ -596,6 +596,24 @@ def add_comment(request):
     return JsonResponse({"status": True})
 
 
+def get_users(request):
```

Comment:
Ah, I had forgotten about this and wanted to discuss options with you. I originally added the AJAX decorator, but it returns `status 400`.

The login decorator does not appear to cause any issues.

Reply:
Yeah, [there](https://github.com/mozilla/pontoon/pull/1623/files?file-filters%5B%5D=.css&file-filters%5B%5D=.json&file-filters%5B%5D=.lock&file-filters%5B%5D=.py#diff-54ad6848e541cb011eb00575cfbfbb8cR28) is a `_` instead of `-`.